src/appkit.cs

```diff
@@ -14851,9 +15154,11 @@ partial interface NSView : NSDraggingDestination, NSAnimatablePropertyContainer,
 		bool LockFocusIfCanDraw ();
 
 		[Export ("lockFocusIfCanDrawInContext:")]
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use 'NSView.DisplayRectIgnoringOpacity (CGRect aRect, NSGraphicsContext context)' to draw a view subtree into a graphics context.")]
```

Comment:
Minor: no specific rule here but I think in general we don't put the arguments variables.

So `NSView.DisplayRectIgnoringOpacity (CGRect, NSGraphicsContext)`. As in the method that take a `CGRect` and a `NSGraphicsContext`.